core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -134,7 +134,11 @@ def _fit(self,
              n_repeat_start=0,
              groups=None,
              **kwargs):
-        use_child_oof = self.params.get('use_child_oof', False)
+        # HACK to be able to change these variables at fit time
+        self.params['use_child_oof'] = kwargs.get('use_child_oof', self.params.get('use_child_oof', False))
+        self.params['save_bag_folds'] = kwargs.get('save_bag_folds', self.params.get('save_bag_folds', True))
```

Comment:
why can't these just be passed into fit() thru `params`?

Reply:
It's because we don't know which model will be used as the proxy model until validation score results are returned. What do you think about me just setting `self.params['use_child_oof'] = False` in FeatureSelector instead of putting this logic in `fit`? Still hacky but not as much.